### PR TITLE
add dependencies and python version to toml

### LINKED ISSUE #5: Python version?
Does this require Python 2.X or 3.X?

### PR DESCRIPTION
**Pull request by wooshaun53**

### CODE REVIEW

File: src/example.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+# Create a 2-D array, set every second element in
+# some rows and find max per row:
+
+x = np.arange(15, dtype=np.int64).reshape(3, 5)
+x[1:, ::2] = -99
+x
```
Comment (zhthean) [inline src/example.py:8]: Is this needed for this script?

File: src/example.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# array([[  0,   1,   2,   3,   4],
+#        [-99,   6, -99,   8, -99],
+#        [-99,  11, -99,  13, -99]])
+
+x.max(axis=1)
```
Comment (zhthean) [inline src/example.py:13]: Same

**Review by zhthean — APPROVED** [review]

File: src/example.py (hunk 1 above)
Comment (wooshaun53) [reply src/example.py:8]: Nope but nice catch. I just copy pasted the entire numpy example blindly.
Will remove after I found or write a better example

File: src/example.py (hunk 2 above)
Comment (wooshaun53) [reply src/example.py:13]: Nope but nice catch. I just copy pasted the entire numpy example blindly.
Will remove after I found or write a better example

**Pull request merged**